@navikt/core/react/src/accordion/AccordionContent.tsx

```diff
@@ -16,6 +16,7 @@ const AccordionContent = forwardRef<HTMLDivElement, AccordionContentProps>(
     const context = useContext(AccordionItemContext);
 
     const { cn } = useRenameCSS();
+    const themeContext = useThemeInternal(false);
```

Comment:
Since this context has a default value now, I think we can do this:

```suggestion
    const themeContext = useThemeInternal();
```

And then we don't need the optional chaining.